packages/xo-web/src/common/xo/index.js

```diff
@@ -1207,8 +1207,33 @@ export const copyVms = (vms, type) => {
         _vms.map((vm, index) => cloneVm({ id: vm }, false, names[index]))
       )
     }
-
     if (sr !== undefined) {
+      if (process.env.XOA_PLAN < 3) {
+        const promise = [_call('xo.getAllObjects', { filter: { id: sr } })]
+        _vms.map(id =>
+          promise.push(_call('xo.getAllObjects', { filter: { id: id } }))
+        )
+        let isLocal = true
+        Promise.all(promise).then(response => {
+          let ipPool
+          response.map(objects => {
+            forEach(objects, object => {
+              ipPool === undefined || ipPool === object.$poolId
+                ? (ipPool = object.$poolId)
+                : (isLocal = false)
+            })
+          })
+
+          !isLocal
+            ? error(_('copyVm'), _('upgradeNeeded'))
```

Comment:
What about using the message `_('availableIn')` instead? Because the users will not know if it refers to the XOA version or the XOA plan.